packages/extension-touchscreen-buttons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jspsych-contrib/extension-touchscreen-buttons",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
This should not be manually updated. The GitHub action will update this based on the change set.